packages/driver/src/cypress/utils.ts

```diff
@@ -399,4 +403,28 @@ export default {
   isPromiseLike (ret) {
     return ret && _.isFunction(ret.then)
   },
+
+  /* Given a chain of functions, return the actual subject. `subjectChain` might look like any of:
+   * [<input>]
+   * ['foobar', f()]
+   * [undefined, f(), f()]
+   */
+  getSubjectFromChain (subjectChain: SubjectChain, cy) {
+    // If we're getting the subject of a previous command, then any log messages have already
```

Comment:
Shouldn't the existing log messages be updated with the new subject value so it's reflected correctly in the console log when clicking a command in the reporter?

Reply:
I don't think we do. Imagine if you do something like this:

```
cy.get('button').as('loginButton').click()

...stuff happens, the page navigates, more commands are run...

cy.get('@loginButton').should('not.exist')
```

The command log would look like:

```
- get
- click

...other stuff...

- get
  - should
```
We wouldn't want to update the first log message with the now non-existent button while executing `.get('@loginButton')` - we'd want to leave it intact, as it was when the command resolved, even if we later invoke get getFn() (to get the subject for the second .get()).